src/components/TabButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import Link from "next/link";
+import { usePathname, useSelectedLayoutSegment } from "next/navigation";
+
+interface TabButtonsProps {
+  queries: string[];
+}
+
+const TabButtons = ({ queries }: TabButtonsProps) => {
+  const pathname = usePathname() ?? "";
```

Comment:
nit: Use camel case.

Reply:
do you mean making pathname into pathName?